website/src/components/features.tsx

```diff
@@ -0,0 +1,115 @@
+import { Tab } from "@headlessui/react";
```

Comment:
We currently have material UI and now headlessui, do you like headless UI better?

Reply:
I am suggesting we move from material UI to browser defaults. HeadlessUI works well with tailwind and provides only components that dont come default in html. Popovers, Tabs etc.